internal/util/importutilv2/numpy/util.go

```diff
@@ -240,7 +240,7 @@ func convertNumpyType(typeStr string) (schemapb.DataType, error) {
 		return schemapb.DataType_Int16, nil
 	case "i4", "<i4", "|i4", ">i4", "int32":
 		return schemapb.DataType_Int32, nil
-	case "i8", "<i8", "|i8", ">i8", "int64":
+	case "i8", "<i8", "|i8", ">i8", "int64": // todo: how to handle Timestamptz?
```

Comment:
We will no longer maintain the import of numpy. There is no need to implement it for now.